### PR TITLE
Feature/add media player

### PR DESCRIPTION
**Pull request by govindappaarun**
- add react player to play videos 
- add thumbnails and player

### CODE REVIEW

File: src/pages/playlists/viewPlaylist.js [hunk 1]
```diff
@@ -5,11 +5,7 @@ import userService from "src/services/userService";
 import styled from "styled-components";
```
Comment (deekshithmd) [inline src/pages/playlists/viewPlaylist.js:5]: In the application...after adding one playlist it is not possible to add another playlist

File: src/pages/video/video.js [hunk 2]
```diff
@@ -39,7 +40,7 @@ export default function VideoPage() {
         <StyledMain>
           <VideoCard>
             <div className="card-media">
-              <img src="https://picsum.photos/300/400/" alt="video" />
+              <ReactPlayer url={`https://www.youtube.com/watch?v=${id}`} />
             </div>
           </VideoCard>
           <Details display="flex" gap="sm">
```
Comment (deekshithmd) [inline src/pages/video/video.js:46]: Better to add watchlater and add to playlist functionality in single video page

**Review by deekshithmd — APPROVED** [review]

File: src/pages/video/video.js (hunk 2 above)
Comment (govindappaarun) [reply src/pages/video/video.js:46]: Ok, will check this

File: src/pages/playlists/viewPlaylist.js (hunk 1 above)
Comment (govindappaarun) [reply src/pages/playlists/viewPlaylist.js:5]: I tested in preview, it seems to work fine with me

Comment (deekshithmd) [reply src/pages/playlists/viewPlaylist.js:5]: ok...

**Pull request merged**